src/table/utils/handle-key-press.ts

```diff
@@ -180,22 +180,25 @@ export const handleBodyKeyDown = ({
     bottom: isSelectionMode && totalsPosition === 'bottom' ? 1 : 0,
   };
 
+  const basicFeaturesEnabled = isFlagEnabled('PS_15585_SN_TABLE_BASIC_FEATURES');
+
   switch (evt.key) {
     case 'ArrowUp':
```

Comment:
Seems we are using this strings (arrow keys) in many places, might be worth creating an enum and have them only in one place.